interface/VideoDecoderDefs.h

```diff
@@ -83,6 +84,7 @@ typedef struct {
     int32_t bitrate;
     int32_t framerateNom;
     int32_t framerateDenom;
+    uint32_t fourcc;
```

Comment:
Do we need to increment the libyami version with this interface data structure change?